{
  "comments": [
    {
      "key": {
        "uuid": "b2c29f45_dabb9b42",
        "filename": "quads/api_v2.py",
        "patchSetId": 2
      },
      "lineNbr": 264,
      "author": {
        "id": 1004250
      },
      "writtenOn": "2020-03-31T08:21:31Z",
      "side": 1,
      "message": "nice fix though a better place to have this is under the prep_data method (called on line #208 here) on model.py for the Cloud Document.",
      "range": {
        "startLine": 264,
        "startChar": 28,
        "endLine": 264,
        "endChar": 36
      },
      "revId": "72be2b5ab1257aa852e234001c45e7d0848c87a2",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}